openpype/hosts/openrv/api/ocio.py

```diff
@@ -0,0 +1,115 @@
+"""Helper functions to apply OCIO colorspace settings on groups.
+
+This tries to set the relevant OCIO settings on the group's look and render
+pipeline similar to what the OpenColorIO Basic Color Management package does in
+OpenRV through its `ocio_source_setup` python file.
```

Comment:
It would be great if we could also autoload this package - but I know too little of RV to say what the best approach is to do so.

Reply:
I've asked more information about this on the Shotgrid Software community about [Autoload OpenColorIO Basic Color Management RV package](https://community.shotgridsoftware.com/t/autoload-opencolorio-basic-color-management-rv-package/17239/1).